packages/components/src/components/app-linked-with-wallet-icon/app-linked-with-wallet-icon.scss

```diff
@@ -10,6 +10,7 @@
         position: absolute;
         top: 0;
         left: 0;
+        z-index: 1;
```

Comment:
@hamid-deriv is this a good practice? The reason I am asking this is because this may cause the `top-icon` to overlap when it is used inside of a scrollable parent. Had similar issue in `wallet-card`.

Reply:
I don't think that this makes a problem as I use `position: relative` on its parent.